tests/compatibility-test.py

```diff
@@ -7,17 +7,20 @@
 import string
 import docker
 
-from kuberay_utils import utils
+import kuberay_utils.utils as utils
 from framework.prototype import (
-    CONST,
     CurlServiceRule,
-    K8S_CLUSTER_MANAGER,
-    OperatorManager,
     RuleSet,
-    shell_subprocess_run,
     show_cluster_info
 )
 
+from framework.utils import (
```

Comment:
What's the difference between `framework.utils` and `kuberay_utils.utils`?
Does each file have a different purpose or does each contain an arbitrary collection of utility functions?

Reply:
`kuberay_utils.utils` is only for `compatibility-test`. For example, the function [create_ray_cluster](https://github.com/ray-project/kuberay/blob/c983eac3f4b1eee0817ccba7141a61818d0c5fcc/tests/kuberay_utils/utils.py#L38) will create a RayCluster and a NodePort service. Hence, we cannot use `RayClusterAddCREvent` directly. We still need some PRs to totally remove the old test framework.

Reply:
I see, so kuberay_utils is the "legacy utils file".
Make sense!